scripts/waypoints_test.py

```diff
@@ -22,7 +21,7 @@ def constrain(x, l, h):
     safety_client = SafetyClient('motion_command_coordinator')
     assert safety_client.form_bond()
 
-    velocity_pub = rospy.Publisher('movement_velocity_targets', TwistStampedArray, queue_size=0)
+    motion_point_pub = rospy.Publisher('motion_point_targets', MotionPointStampedArray, queue_size=0)
```

Comment:
What subscribes to this and are there any other PR's that this depends on?

Reply:
Low level motion. Nothing depends on this but the whole PR depends on one that doesnt exist yet.

Reply:
What does this pr depend on? Another topic or node? I thought you made a motion point transcriber node.

Reply:
It depends on a hlm outputting motion points with z position filled out.

Reply:
Velocity commands require a z position now?